modules/KalturaSupport/resources/mw.id3Tag.js

```diff
@@ -71,17 +71,26 @@
         parseTag: function(tag){
             var time;
             if ( tag ) {
+                // on IE the tag needs to be parsed
                 time = tag.timestamp / 1000;
+                if(!time){
```

Comment:
This is wrong... 
Fixed the id3 tag and the correct place:
https://github.com/kaltura/mwEmbed/blob/db76a29d15909ba3f85fb65108f9ff302c88d900/modules/EmbedPlayer/resources/mw.EmbedPlayerSilverlight.js#L692